src/main/java/cam72cam/prospectus/Config.java

```diff
@@ -0,0 +1,72 @@
+package cam72cam.prospectus;
+
+import net.minecraftforge.common.config.Configuration;
+import org.apache.logging.log4j.Level;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Config {
```

Comment:
Could you switch this to use net.minecraftforge.common.config.Config instead?  It's the more modern way of doing it and supports in-game mod config.

See: https://github.com/cam72cam/WinterWonderLand/blob/master/src/main/java/cam72cam/winterwonderland/Config.java#L5
for an example

Reply:
:+1: